h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -147,6 +180,8 @@ public boolean getUseExtendedOutput() {
    */
   public EasyPredictModelWrapper(Config config) {
     m = config.getModel();
+    // Ensure an error consumer is always instantiated to avoid missing null-check errors.
+    errorConsumer = config.getErrorConsumer() == null ? new VoidErrorConsumer() : config.getErrorConsumer();
```

Comment:
Making sure there is always one errorConsumer to avoid forgotten null checks. For detailed info see VoidErrorConsumer.

Reply:
This is both good and bad thing - you lose the ability to check whether the consumer is defined.

Thinking of analogy of logging: you won't have method "isDebugEnabled" and you won't be able to safeguard more time-consuming operations into an if statement.

    if (errorConsumer != null) {
       // make a nicer message
    }

Reply:
In future, the Void consumer can return false in method name e.g. isEnabled() ... all others would return true :)